token-lending/js/package.json

```diff
@@ -37,6 +37,7 @@
     "@solana/spl-token": "0.1.3",
     "@solana/web3.js": "^1.7.0",
     "bn.js": "^5.2.0",
+    "keccak": "^3.0.1",
```

Comment:
Where is this used?

Reply:
If we don't have this it will report:
```
Error: Cannot find module 'keccak'
Require stack:
- /Users/wangge/Projects/rust/solana-program-library/token-lending/js/node_modules/@solana/spl-token/node_modules/@solana/web3.js/lib/index.cjs.js
- /Users/wangge/Projects/rust/solana-program-library/token-lending/js/node_modules/@solana/spl-token/lib/index.cjs.js
- /Users/wangge/Projects/rust/solana-program-library/token-lending/js/cli/token-lending-test.ts
- /Users/wangge/Projects/rust/solana-program-library/token-lending/js/cli/main.ts
    at Function.Module._resolveFilename (node:internal/modules/cjs/loader:923:15)
    at Function.Module._load (node:internal/modules/cjs/loader:768:27)
    at Module.require (node:internal/modules/cjs/loader:995:19)
    at require (node:internal/modules/cjs/helpers:92:18)
    at Object.<anonymous> (/Users/wangge/Projects/rust/solana-program-library/token-lending/js/node_modules/@solana/spl-token/node_modules/@solana/web3.js/lib/index.cjs.js:26:24)
    at Module._compile (node:internal/modules/cjs/loader:1091:14)
    at Object.Module._extensions..js (node:internal/modules/cjs/loader:1120:10)
    at Module.load (node:internal/modules/cjs/loader:971:32)
    at Function.Module._load (node:internal/modules/cjs/loader:812:14)
    at Module.require (node:internal/modules/cjs/loader:995:19)
npm ERR! code 1
npm ERR! path /Users/wangge/Projects/rust/solana-program-library/token-lending/js
npm ERR! command failed
npm ERR! command sh -c ts-node cli/main.ts

npm ERR! A complete log of this run can be found in:
npm ERR!     /Users/wangge/.npm/_logs/2021-05-02T05_15_52_008Z-debug.log

```

Looks like a missing dependency on the `spl-token` program?